test/MarketMaker.js

```diff
@@ -191,7 +171,7 @@ contract('marketMaker', function(accounts) {
     data.marketMakerBurnedTokens.should.be.bignumber.equal(await mm.totalBurnedTokens.call());
 
     if (data.marketMakerMonth < periods) {
-      data.marketMakerBuyPrice.should.be.bignumber.equal(await mm.getBuyPrice());
+      data.marketMakerBuyPrice.should.be.bignumber.equal(await mm.getBuyPrice(), 0, BigNumber.ROUND_DOWN);
```

Comment:
Same here, what's the reason for rounding here?

Reply:
It was WIP, I was working on teh last tests of market maker, the rounding is removed now :)